### ISSUE TITLE
Feature: Sign 'WireMock.Net'

### ISSUE THREAD

**Issue by icraftsoftware** [WireMock-Net/WireMock.Net#160]
Hello,

I have no choice, I am working on BizTalk Server integration solutions, and as you know, BTS requires the assemblies to be GACed and a fortiori signed. Because all my projects are signed, I can't use WireMock. Please, provide a signed version of your assembly; on nuget.org as well. It would help all of us that must use signed assemblies, and would not bother anyone else.

Thank you

François

**Comment by StefH** [comment]
At this moment WireMock.Net cannot be strong name signed because `RestEase` is not yet signed.

Maybe you can use this project https://github.com/dsplaisted/strongnamer ?

**Comment by icraftsoftware** [comment]
Never heard of StrongNamer, but unfortunately it did not work. What did the trick was the [following Stack Overflow solution](https://stackoverflow.com/a/46631067/1789441), see the comment I made (incl. 4 other comments that show the command lines to use).

I also had to make sure that no two different versions of a given NuGet package were used; this was the case for Newtonsoft.Json.

The main downside is that I cannot blindly restore NuGet packages to build my solution anymore.

Could not you just provide an equivalent to what I've done on NuGet?
